msbuild/Xamarin.MacDev.Tasks/Tasks/CreateBindingResourcePackageBase.cs

```diff
@@ -37,6 +37,13 @@ public override bool Execute ()
 				return true;
 			}
 
+			var inexistent = NativeReferences.Where (v => !(Directory.Exists (v.ItemSpec) || File.Exists (v.ItemSpec)));
+			if (inexistent.Any ()) {
+				foreach (var inex in inexistent)
+					Log.LogError (MSBStrings.E0190 /* The NativeResource item '{0}' does not exist. */, inex.ItemSpec);
```

Comment:
lgtm but i think nonexistent sounds better
```suggestion
			var nonexistent = NativeReferences.Where (v => !(Directory.Exists (v.ItemSpec) || File.Exists (v.ItemSpec)));
			if (nonexistent.Any ()) {
				foreach (var nonex in nonexistent)
					Log.LogError (MSBStrings.E0190 /* The NativeResource item '{0}' does not exist. */, inex.ItemSpec);
```